package.json

```diff
@@ -17,5 +17,8 @@
         "lodash": "^4.17.4",
         "tailwindcss": ">=0.5.2",
         "vue": "^2.5.7"
+    },
+    "dependencies": {
+        "moment": "^2.22.2"
```

Comment:
This could probably be moved to `devDependencies` since we're building locally/committing compiled assets

Reply:
Good catch @sbine